GliaWidgets/Sources/ViewModel/Call/CallViewModel.swift

```diff
@@ -323,12 +323,12 @@ extension CallViewModel {
     private func showLocalVideo(with stream: CoreSdkClient.VideoStreamable) {
         action?(.switchToVideoMode)
         action?(.setLocalVideo(stream.getStreamView()))
-        stream.onHold = { isOnHold in
+        stream.onHold = { [weak stream] isOnHold in
             // onHold case is not handled deliberately because it
             // is handled elsewhere. This logic may need some adjustments
             // in future
             if !isOnHold {
-                stream.playVideo()
+                stream?.playVideo()
```

Comment:
Should there be a log in case the stream has been deallocated? Could be useful for debugging maybe?

Reply:
I think it makes sense. But since we agreed on logs consistency between  mobile platforms, this has to be done in separate tickets for both platforms.